contracts/utils/Address.sol

```diff
@@ -57,4 +57,42 @@ library Address {
         (bool success, ) = recipient.call{ value: amount }("");
         require(success, "Address: unable to send value, recipient may have reverted");
     }
+
+    /**
+     * @dev Replacement for Solidity's low-level `call`: performs a low-level
```

Comment:
Instead of repeating the same docs here, we would put a link to the other one ("See `functionCall`") and only explain what's different about this one (in this case the default revert reason).

I'm not sure how we can automatically generate a link to a function with an overload like this one. @nventuro do you remember?

Reply:
What I do is just run the docsite manually and copy-paste the identifier. I can add this myself on top of this PR later on.

Reply:
Yes. I repeated the docs because I saw it was done a similar approach in the [div](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/math/SafeMath.sol#L91-L125) example. But I agree better not to repeat.

Reply:
Ah that's ineteresting, thanks. We should improve those.